tools/kagami/src/docs.rs

```diff
@@ -29,8 +29,8 @@ where
 {
     fn get_markdown<W: Write>(writer: &mut W) -> color_eyre::Result<()> {
         let Value::Object(docs) = Self::get_docs() else {
-                unreachable!("As top level structure is always object")
-            };
+            unreachable!("As top level structure is always object")
```

Comment:
```suggestion
            unreachable!("As top level structure is always an object")
```

Should be more correct?